java/odsa-spring/src/main/resources/application.properties

```diff
@@ -0,0 +1,21 @@
+# https://www.oracle.com/database/technologies/getting-started-using-jdbc.html
+spring.datasource.url=<URL>
+spring.datasource.username=<USERNAME>
+spring.datasource.password=<PASSWORD>
+
+# Properties for UCP - Universal Connection Pool (UCP). Spring Boot 2.4.0 or above is required
+spring.datasource.driver-class-name=oracle.jdbc.OracleDriver
+spring.datasource.type=oracle.ucp.jdbc.PoolDataSource
+spring.datasource.oracleucp.connection-pool-name=connectionPoolName1
+spring.datasource.oracleucp.initial-pool-size=15
```

Comment:
Do you need that many initial connections for this demo use case?

Reply:
Hi Kuassi, spot on! Not at all, I copied that verbatim from your OCW/J1 deck. Changed it to the values below as this is for educational purposes only indeed, so it's fixed now. Thanks!

spring.datasource.oracleucp.initial-pool-size=1
spring.datasource.oracleucp.min-pool-size=1
spring.datasource.oracleucp.max-pool-size=2